ucte/ucte-network/src/main/java/com/powsybl/ucte/network/io/UcteReader.java

```diff
@@ -280,12 +273,16 @@ private void parseRecords(UcteRecordParser parser, UcteNetwork network) throws I
     }
 
     public UcteNetwork read(BufferedReader reader) throws IOException {
+
+        Reporter readReporter = reporter.createChild("UcteReading", "Reading UCTE network file");
```

Comment:
this kind of key is part of the public API for consummers of these logs to extract and manipulate those logs ? if so, maybe make it a constant and document it, instead of directly using a string in the code which is not very easy to find?

Reply:
Indeed, we could define the value and taks keys as constants in class files (or in resource files?). But to keep the modularity of powsybl, the keys for one process should be in the corresponding artifact only. There won't be a unique file, unfortunately, which makes it a bit less interesting.

About documenting all the keys I think it would be a bit heavy, and redundant with the default message, don't you think? Maybe add the default message next to the keys constant? but then reading the report code would be unclear.

Reply:
I see what you mean, but when you are writing code to parse these reports, I think you want a definitive spec of what can be in the report, not just looking at one report file (which might not represent all possible reports) to guess things no ?

When I said document, I didn't mean to describe each and every key in the documentation, but at least list them. But you are right that the default message is a very good documentation, so having them together is a good idea. I think the report good would still be clear. Not sure what's the best way to group them together though, constants with similar names ? Is an enum better ? something else ?

I don't think we need a view of all possible report keys. We you want to parse an ucteimport report, you go and read the ucte documentation for its keys and you don't care about others ?
But if we wanted, we could do something with serviceloader to discover all the things that that write reports in the current jvm and ask them to give the list of keys, but not sure if it's useful.